StreamChatClient_v3/WebSocketClient/Events/EventDecoder.swift

```diff
@@ -54,7 +54,7 @@ protocol AnyEventDecoder {
 extension EventDecoder: AnyEventDecoder {}
 
 /// The DTO object mirroring the JSON representation of the event.
-struct EventResponse<ExtraData: ExtraDataTypes>: Decodable {
+struct EventPayload<ExtraData: ExtraDataTypes>: Decodable {
     let connectionId: String?
     
     let channelPayload: ChannelPayload<ExtraData>?
```

Comment:
Can we rename these to drop the `payload` suffix? With the introduction of `EventWthPayload,` seems redundant to write `payload.XPayload`

Reply:
Makes sense 👍